Userland/Libraries/LibMedia/FFmpeg/FFmpegHelpers.h

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2024, Gregory Bertilson <zaggy1024@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <LibMedia/CodecID.h>
+
+extern "C" {
+#include <libavcodec/avcodec.h>
```

Comment:
ew, does this header really not have extern "C" if __cplusplus in it automagically?

Reply:
Unfortunately, it appears not. I get undefined references to mangled names without this.